ci/install.sh

```diff
@@ -19,7 +19,7 @@ echo "Install neovim module for Python 2."
 CC=cc python2 -m pip -q install --user --upgrade pynvim
 
 echo "Install neovim RubyGem."
-gem install --no-document --pre neovim
+gem install --no-document --user-install --pre neovim
```

Comment:
Ruby tests are skipped. Either update `PATH`, run sudo, or setup rvm,rbenv,chruby.

```
/usr/local/lib/site_ruby/2.5.0/rubygems/installer.rb:715: warning: Insecure world writable dir /home/runner in PATH, mode 040777
WARNING:  You don't have /home/runner/.gem/ruby/2.5.0/bin in your PATH,
	  gem executables will not run.
```

Reply:
> Ruby tests are skipped. Either update `PATH`, run sudo, or setup rvm,rbenv,chruby.

[Fixed](https://github.com/neovim/neovim/pull/13297).  Thanks for noticing.